server/src/main/java/org/apache/iotdb/db/metadata/MManager.java

```diff
@@ -313,23 +279,52 @@ private int applyMlog(MLogReader mLogReader) {
     return idx;
   }
 
+  private void checkMTreeModified() {
+    if (logWriter == null || logFile == null) {
+      // the logWriter is not initialized now, we skip the check once.
+      return;
+    }
+    if (System.currentTimeMillis() - logFile.lastModified() < mtreeSnapshotThresholdTime) {
+      if (logger.isDebugEnabled()) {
```

Comment:
Good Job. I have a little question.
this bransh only for print log? 
I fell if only for print log , That's not necessary.

Reply:
The mtreeModifiedCheck has already been simplified in the commit "[IOTDB-1498] Simplify checkMTreeModified in MManager".

Reply:
Fixed